Update insider-risk-management-solution-overview.md (#9555)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/insider-risk-management-solution-overview.md

**Insider risk management workflow**:
>[!VIDEO https://www.microsoft.com/videoplayer/embed/RE4OUXB]

## Subscriptions and licensing

Before you get started with insider risk management, you should confirm your [Microsoft 365 subscription](https://www.microsoft.com/microsoft-365/compare-all-microsoft-365-plans) and any add-ons. To access and use insider risk management, your organization must have one of the following subscriptions or add-ons:

- Microsoft 365 E5/A5/F5/G5 subscription (paid or trial version)
- Microsoft 365 E3/A3/F3/G3 subscription + the Microsoft 365 E5/A5/F5/G5 Compliance add-on
- Microsoft 365 E3/A3/F3/G3 subscription + the Microsoft 365 E5/A5/F5/G5 Insider Risk Management add-on
- Office 365 E3 subscription + Enterprise Mobility and Security E3 + the Microsoft 365 E5 Compliance add-on

> Users included in insider risk management policies must be assigned one of the licenses above.

Reviewer: ```suggestion
Users included in insider risk management policies must be assigned one of these licenses.
```
